apps/web/components/molecules/tabs/tabs.tsx

```diff
@@ -0,0 +1,55 @@
+import { Tab } from '@headlessui/react';
+import { Opinions } from 'molecules/opinions/opinions';
+import clsx from 'clsx';
+import type { ReactNode } from 'react';
+import { Comments } from 'molecules/comments/comments';
+
+type TabObject = {
+	label: string;
+	content: ReactNode;
+};
+
+export const Tabs = () => {
+	const tabs: TabObject[] = [
+		{
+			label: 'Opinie',
+			content: <Opinions />,
+		},
+		{
+			label: 'Rozwiązania',
+			content: 'Tab 2',
+		},
+		{
+			label: 'Komentarze',
+			content: <Comments />,
+		},
+	];
```

Comment:
Wyniósł bym to na zewnątrz komponentu i content przekazywałbym jako content: Comments bez wywołania komponentu. Wtedy w mapowaniu robiłbyś const Content = tab.content i <Content />